autodeploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# There are two npm versions used in travis (.travis.yml)
+# and only one deploy is needed 
+if [ "$TRAVIS_NODE_VERSION" != "5" ]; then
+	echo "Skipping deploy for npm version != 5"
+	exit 0
+fi
+
+# Just for the PR: autodeploy branch rather than master
+#SOURCE_BRANCH="master"
+SOURCE_BRANCH="autodeploy"
```

Comment:
Don't forget to switch back to `master` after you verified that it still works after your changes.

(this also serves as a reminder for reviewers to check that this has been done before merging the PR)